insights/parsers/filefrag.py

```diff
@@ -0,0 +1,40 @@
+"""
+Filefrag - Command ``/sbin/filefrag /boot/grub2/grubenv``
```

Comment:
Just like the docstings of class `Filefrag`, what about 
```suggestion
Filefrag - Command ``/sbin/filefrag <specified files>``
```